app/commands/tooling_job/process.rb

```diff
@@ -40,6 +40,20 @@ def call
           JSON.parse(download_file(attrs["output"]["analysis.json"]))
         )
       end
+
+      client.update_item(
+        table_name: Exercism.config.dynamodb_tooling_jobs_table,
+        key: { id: id },
+        expression_attribute_names: {
+          "#JS": "job_status",
+          "#LU": "locked_until"
+        },
+        expression_attribute_values: {
+          ":js": "processed",
+          ":lu": nil
```

Comment:
I've set the `locked_until` property to `nil`. Is that the right thing to do, or do you want to leave it unchanged?

Reply:
```suggestion
```